cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml

```diff
@@ -64,6 +64,15 @@ spec:
             - name: usr-ca-certs
               mountPath: /usr/share/ca-certificates
               readOnly: true
+        - name: prometheus-to-sd-exporter
+          image: gcr.io/google-containers/prometheus-to-sd:v0.2.1
+          command:
+            - /monitor
+            - --source=heapster:http://localhost:8082?whitelisted=stackdriver_requests_count,stackdriver_timeseries_count
+            - --stackdriver-prefix=container.googleapis.com/internal/addons
```

Comment:
Considering that this sidecar monitors only heapster, maybe we also should pass pod_id and namespace_id here?